discovery_engine/lib/src/domain/event_handler.dart

```diff
@@ -15,6 +15,7 @@
 import 'dart:typed_data' show Uint8List;
 import 'package:async/async.dart' show StreamGroup;
 import 'package:hive/hive.dart' show Hive;
+import 'package:meta/meta.dart';
```

Comment:
Mmh, if this is a must for us we should probably get a linting rule going. Given that IntelliJ auto-imports all of these, otherwise I will definitely keep running into them 😅

Reply:
yeah, would be nice if such rule existed, or if we could add custom rules, but unfortunately it's not the case, sorry